source/globalCommands.py

```diff
@@ -3630,8 +3630,8 @@ def script_reloadPlugins(self, gesture):
 
 	@script(
 		description=_(
-			# Translators: input help mode message for Report destination URL of navigator link command
-			"Report the destination URL of the link in the navigator object. "
+			# Translators: input help mode message for Report destination URL of a link command
+			"Report the destination URL of the link at the current user position. "

```

Comment:
"user position" is a vague concept.

```suggestion
			"Report the destination URL of the currently focused link or graphic."

```

Reply:
I've rephrased the description slightly, to consistently mention caret / focus everywhere these commands are documented.